.kokoro/trampoline_v2.sh

```diff
@@ -159,7 +159,7 @@ if [[ -n "${KOKORO_BUILD_ID:-}" ]]; then
 	"KOKORO_GITHUB_COMMIT"
 	"KOKORO_GITHUB_PULL_REQUEST_NUMBER"
 	"KOKORO_GITHUB_PULL_REQUEST_COMMIT"
-	# For FlakyBot
+	# For Build Cop Bot
```

Comment:
out-of-date. Maybe needs https://github.com/googleapis/python-datacatalog/pull/120 first